src/template/partials/header.html

```diff
@@ -0,0 +1,54 @@
+<header>
+   
+                    
+        <nav class="navbar navbar-expand-lg bg-body-tertiary">
+            <div class="container">
+                
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon">
+                        <img src="assets/images/bar.svg" alt="bar">
+                    </span>
+                </button>
+
+                <h1>
+                <a class="navbar-brand" href="#header-sec">
+                    <img class="d-none d-md-block" src="assets/images/LOGO.png" alt="logo">
+                    <img class="d-md-none" src="assets/images/LOGO-sm.png" alt="logo">
```

Comment:
Why did you duplicate the logo, you should control the size with width or height and I prefer the height

Reply:
done